infrastructure/smart-contracts/test/tinykeys/RefereePoolSubmissions.mjs

```diff
@@ -386,7 +387,7 @@ function PoolSubmissionPermissions(deployInfrastructure) {
         await submitTestChallenge(referee, challenger, startingAssertion, winningStateRoot);
 
         // Create a pool with the minted keys
-        const poolAddress = await createPool(poolFactory, addr1, keys);
+        const poolAddress = await createPool(poolFactory, addr1, winningKeys);
```

Comment:
Does it matter if stake a winning key to the pool ?

Reply:
No, but keys was not defined so the test was failing.